public/v4/apps/mayan-edms.yml

```diff
@@ -0,0 +1,158 @@
+
+        - id: $$cap_redis_pass
+          label: Redis password
+          defaultValue: mayanredispassword
+          validRegex: /^([^\s^\/])+$/
+
+        - id: $$cap_dbname
+          label: Database Name
+          defaultValue: 'mayan'
+          validRegex: /^([^\s^\/])+$/
+
+        - id: $$cap_dbuser
+          label: Database User
+          defaultValue: 'mayandbuser'
+          validRegex: /^([^\s^\/])+$/
+
+        - id: $$cap_dbpass
+          label: Database Password
+          defaultValue: $$cap_gen_random_hex(64)
+          validRegex: /^([^\s^\/])+$/
+
+        - id: $$cap_default_document_lang
+          label: Default document language
+          defaultValue: eng
+          description: Use the ISO 639-3 code (https://en.wikipedia.org/wiki/ISO_639). Will default to *eng* if left blank.
+
+        - id: $$cap_language_code
+          label: Default language for the installation
+          defaultValue: en-us
+          description: Mayan EDMS UI will default to this language if the user's locale is not found. Will default to *en-us* if left blank
+
+        - id: $$cap_ocr_langs
+          label: Extra packages
+          defaultValue: 'tesseract-ocr-spa'
+          description: Use to add extra languagues for OCR (English is installed by default). Insert as many packages as you need, separated by a space, as *tesseract-ocr-CODE*. Language codes in https://tesseract-ocr.github.io/tessdoc/Data-Files-in-different-versions.html
+
+        - id: $$cap_timezone
+          label: Time Zone
+          defaultValue: UTC
+          description: Leave blank if you'd rather change it in System => Setup => Settings
+
+        - id: $$cap_username
+          label: Admin username
+          defaultValue: admin
+          description: Username for your first login
+
+        - id: $$cap_email
+          label: Admin email
+          defaultValue: example@example.com
+
+        - id: $$cap_pass
+          label: Admin password
+          defaultValue: $$cap_gen_random_hex(16)
+          validRegex: /^([^\s^\/])+$/
+          description: It will be prompted to you at startup and can be changed, but copy it just in case.
+
+        - id: $$cap_project_title
+          label: Project Title (Site's name)
+          defaultValue: 'Mayan EDMS'
+          description: Title and header of your site
+
+    instructions:
+        start: >-
+            Mayan EDMS is an opensource web-based Document Management System. 
+            Most settings can be set inside the app, but environment variables will override them.
+            Minimum requirements 2GB RAM and 2vCPUs. Recommended 4GB RAM and 4vCPUs
+        end: >-
+            Done!
+            Your service is available at http://$$cap_appname.$$cap_root_domain
+    displayName: 'Mayan EDMS'
+    isOfficial: false
+    description: Mayan EDMS is an open source web-based Document Management System with many advanced features.
+    documentation: https://www.mayan-edms.com/
```

Comment:
Suggest putting this website in instructions.start as well as it makes it easier for people to understand what this is.

Reply:
Done. Good idea